doc/source/working_with_images/pxe_client_configuration.rst

```diff
@@ -6,13 +6,13 @@ PXE Client Setup Configuration
 .. sidebar:: Abstract
 
    This page provides further information for handling
-   PXE images built with KIWI and references the following
+   PXE images built with {kiwi} and references the following
    articles:
 
    * :ref:`build_pxe`
 
 All PXE boot based deployment methods are controlled by configuration files
-located in :file:`/srv/tftpboot/KIWI` on the PXE server. Such a configuration
+located in :file:`/srv/tftpboot/{kiwi}` on the PXE server. Such a configuration
```

Comment:
I doubt this is correct. This should not be translated to `KIWI-NG`, right?

Reply:
fixed